cardano-cli/src/Cardano/CLI/Legacy/Run/Governance.hs

```diff
@@ -35,6 +36,13 @@ runLegacyGovernanceCmds = \case
     runLegacyGovernanceGenesisKeyDelegationCertificate sbe genVk genDelegVk vrfVk out
   GovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp ->
     runLegacyGovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp
+  GovernanceCreatePoll prompt choices nonce out ->
+    runGovernanceCreatePoll prompt choices nonce out
+  GovernanceAnswerPoll poll ix mOutFile ->
+    runGovernanceAnswerPoll poll ix mOutFile
+  GovernanceVerifyPoll poll metadata mOutFile ->
+    runGovernanceVerifyPoll poll metadata mOutFile
```

Comment:
Call the era-based run functions using `BabbageEraOnlyBabbage` as the era witness.

Reply:
@newhoggy> what is the rule for allowing that? Is that because the caller is from a `Legacy` package and so we are more lenient?

Reply:
Good question.  This command is somewhat a special case.  It was introduced during Babbage as a short term feature that will be obsoleted by Conway, hence it is Babbage only.  Even if in the era-based version of the command we want to extend its lifetime to include Conway (which is still a possibility), the legacy commands are deprecated and will not extend to Conway.  Therefore we know for sure the legacy command is Babbage-only.

And because it is Babbage-only, the legacy command can provide a witness that satisfies the eon type and also says it is Babbage.